.storybook/themes/news.scss

```diff
@@ -911,6 +911,63 @@
 				double-chain-child-item-empty: (
 					display: contents,
 				),
+				gallery: (
+					components: (
+						icon: (
+							fill: var(--text-color),
+							height: var(--global-spacing-4),
+							width: var(--global-spacing-4),
+						),
+						image: (
+							max-height: 75vh,
+						),
+						carousel-controls: (
+							padding: 0 0 var(--global-spacing-2) 0,
+						),
+						carousel-track: (
+							gap: initial,
+							max-width: initial,
+						),
+						carousel-button: (
+							border: none,
+							box-shadow: none,
+							height: var(--global-spacing-4),
+						),
+						carousel-button-enter-full-screen: (
+							padding: 0,
+						),
+						carousel-additional-controls: (
+							display: none,
+						),
+						carousel-button-toggle-auto-play: (
+							padding: 0 var(--global-spacing-4),
+						),
+						carousel-button-next: (
+							background-color: transparent,
```

Comment:
This is causing a regression in other blocks - we will need to define the default at the component token level